app/src/main/java/de/danoeh/antennapod/adapter/NavListAdapter.java

```diff
@@ -220,20 +220,12 @@ public View getView(int position, View convertView, ViewGroup parent) {
             if (position == itemAccess.getSelectedItemIndex()) {
                 txtvTitle.setTypeface(null, Typeface.BOLD);
                 v.getContext().getTheme().resolveAttribute(de.danoeh.antennapod.core.R.attr.drawer_activated_color, typedValue, true);
-                int[] attribute = new int[] { de.danoeh.antennapod.core.R.attr.drawer_activated_color };
-                TypedArray array = v.getContext().obtainStyledAttributes(typedValue.resourceId, attribute);
-                int backgroundResource = array.getColor(0, 0);
-                array.recycle();
-                v.setBackgroundColor(backgroundResource);
```

Comment:
It'd be even better if the logic of getting color from an attribute is extracted as a common helper method.

Reply:
Ya I cleaned this up by using the resourceid instead of getting the color, would you still like to throw this in a helper method? If so would themeutils be a good place to put it?